components/raftstore/src/store/fsm/apply.rs

```diff
@@ -4965,6 +4987,10 @@ mod tests {
         cmd_sink: Option<Arc<Mutex<Sender<CmdBatch>>>>,
         filter_compact_log: Arc<AtomicBool>,
         filter_consistency_check: Arc<AtomicBool>,
+        delay_remove_ssts: Arc<AtomicBool>,
+        last_delete_sst_count: Arc<AtomicU64>,
+        last_pending_clean_sst_count: Arc<AtomicU64>,
+        last_pending_handle_sst_count: Arc<AtomicU64>,
```

Comment:
What do these atomics used for?

Reply:
I think these three fields can be modified by test thread and apply thread, so I'd like use atomics for safety.